Composer/packages/client/src/pages/qna/code-editor.tsx

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/* eslint-disable react/display-name */
+import React, { useState, useEffect, useContext, useMemo } from 'react';
+import { LuEditor, EditorDidMount, defaultQnAPlaceholder } from '@bfc/code-editor';
+import isEmpty from 'lodash/isEmpty';
+import { RouteComponentProps } from '@reach/router';
+import querystring from 'query-string';
+import debounce from 'lodash/debounce';
+import get from 'lodash/get';
+import { CodeEditorSettings } from '@bfc/shared';
+
+import { StoreContext } from '../../store';
+
+interface CodeEditorProps extends RouteComponentProps<{}> {
+  dialogId: string;
+}
+
+const lspServerPath = '/lu-language-server';
+const CodeEditor: React.FC<CodeEditorProps> = (props) => {
```

Comment:
CodeEditor is too generic, can it be QnAEditor?

Reply:
@liweitian Can u see it?